flashinfer/comm/mnnvl.py

```
@@ -484,6 +557,122 @@ def new_mnnvl_memory_address(mapping: Mapping, size: int):
+                            if (
+                                cmsg_level == socket.SOL_SOCKET
+                                and cmsg_type == socket.SCM_RIGHTS
+                            ):
+                                trim = len(cmsg_data) - (
+                                    len(cmsg_data) % fds_arr.itemsize
+                                )
+                                fds_arr.frombytes(cmsg_data[:trim])
+                        if fds_arr:
+                            received_fds[sender_rank] = fds_arr[0]
+                        conn.close()
+                    except Exception as exc:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)